### DISCUSSION TITLE
Movies/Geek themed icons

### DISCUSSION THREAD

**Discussion by lethamburn** [dr-slash/icons-flat-osx#106] [Requests]
Do you have some movies or similar themed icons like the videogame ones? I love the character collection!

**Comment by lucky-yoolk** [comment]
i'm working on this https://yoolk.ninja/iconset/charavatar/ its icons and avatars set from movies and animes.

**Reply by lethamburn** [reply]
Yay! This is so cool! Thank you very much :D
